dataactbroker/file_routes.py

```diff
@@ -179,16 +177,17 @@ def get_fabs_metadata(submission):
         return JsonResponse.create(StatusCode.OK, get_fabs_meta(submission.submission_id))
 
     @app.route("/v1/upload_detached_file/", methods=["POST"])
+    @requires_login
```

Comment:
Should this be just `requires login` or should we be checking for at least `editfabs` permissions?

Reply:
Using `requires_submission_perms` means that a submission already exists to check against, so I added `current_user_can` to `submit_files` and `upload_detached_file`.